modules/tinymce/src/themes/silver/main/ts/ui/core/complex/FontsizeSelect.ts

```diff
@@ -27,6 +27,17 @@ const legacyFontSizes: Record<string, string> = {
   '36pt': '7'
 };
 
+// Note: 'xx-small', 'x-small' and 'large' are rounded to fit our default size list
+const keywordFontSizes: Record<string, string> = {
+  'xx-small': '8pt',
```

Comment:
This looks to be a typo, as I believe this should be `7pt`? At the moment this would cause likely issues, as `xx-small` and `x-small` are using the same sizes.

Reply:
I'm mapping them all to a valid font size in our defaults list, if we mapped `xx-small` to `7pt` it would show as `xx-small` in the menu. However, we could alter the list of available font sizes based on these values so that `7pt` would be available as a choice in the menu

Reply:
We discussed this more on slack, however the gist of it was this was done to try and match our default font-size list. We can't assume others will be using the same, so we need it to fallback to showing the actual size if it can't be mapped to something in the list.